packages/pgsolver/pgsolver.4.4/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis:
+  "A collection of tools for generating, manipulating and - most of all - solving parity games"
+description:
+  "A collection of tools for generating, manipulating and - most of all - solving parity games."
+maintainer: ["Oliver Friedmann" "Martin Lange"]
+authors: ["Oliver Friedmann" "Martin Lange"]
+license: "BSD-3-clause"
+homepage: "https://github.com/tcsprojects/pgsolver"
+bug-reports: "https://github.com/tcsprojects/pgsolver/issues"
+depends: [
+  "ocaml" {>= "4.8"}
+  "dune" {>= "3.10"}
+  "num"
+  "tcs-lib"
+  "extlib"
+  "str"
+  "unix"
```

Comment:
```suggestion
```
Similarly `Str` is the name of a `Stdlib` module (there's no opam-repo package named that),
and like `threads`, `unix` comes with the compiler and can be enabled by passing the right flags.